mod-invoice-storage/schemas/invoice.json

```diff
@@ -137,7 +155,7 @@
     "approvalDate",
     "status",
     "source",
-    "totalAmount",
+    "total",
     "vendorInvoiceNo",
     "voucherNumber",
```

Comment:
Also AFAIK, voucher number is created after invoice is ready for payment, can this be a required field in schema?

Reply:
I agree with you and think that this should be also optional and added by the system once invoice is "Approved".

Reply:
Also looking at [Acquisitions Interface Fields](https://docs.google.com/spreadsheets/d/1F88PoSnEKD7oFnza1ZuOgzAgKnK95XOyTQzxyww3tZA/edit#gid=131223376&range=A28) document I see `payment(voucher)_id` which is also required. I am going to make it optional as well.
@craigmcnally please let us know if you are okay with these changes

Reply:
Yes that makes sense.